src/bin/helpers.js

```diff
@@ -27,6 +27,16 @@ function displayNonInitializedFolderErrorMessage() {
     );
 }
 
+// Check if harmonic.json file exists
+function isHarmonicProject(sitePath) {
```

Comment:
I move this function because of this
```
/Users/viniciusdacal/github/harmonic/src/bin/helpers.js
  36:12  error  isHarmonicProject was used before it was defined  no-use-before-define
```
any suggestions in cases like this?